mock-collector/pom.xml

```diff
@@ -187,4 +187,19 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>java9</id>
+            <activation>
+                <jdk>[1.9,]</jdk>
```

Comment:
This is automatically detected and activated, so it's transparent to the users, they don't need to care about the JDK version

Reply:
yes